### PR TITLE
fix(super-components): url utils

### PR DESCRIPTION
**Pull request by chipp972**
| Question         | Answer
| ---------------- | ---
| Branch?          | `dev-transformation`
| Bug fix?         | no
| New feature?     | yes
| Breaking change? | no
| Tickets          | MANAGER-12395
| License          | BSD 3-Clause

- [x] Try to keep pull requests small so they can be easily reviewed.
- [x] Commits are signed-off
- [ ] ~~Only FR translations have been updated~~
- [x] Branch is up-to-date with target branch
- [x] Lint has passed locally
- [ ] ~~Standalone app was ran and tested locally~~
- [x] Ticket reference is mentioned in linked commits (internal only)
- [ ] ~~Breaking change is mentioned in relevant commits~~

## Related

I added an utils for url on the super-components

### CODE REVIEW

File: packages/super-components/utils/src/url.ts [hunk 1]
```diff
@@ -1,18 +1,75 @@
...
+  | 'GB'
+  | 'ES'
+  | 'PL'
+  | 'PT'
+  | 'IT'
+  | 'DE'
+  | 'IE'
+  | 'QC'
+  | 'WE'
+  | 'WS';
```
[the remark this answers is not in the document]
Comment (chipp972) [reply packages/super-components/utils/src/url.ts:26]: where do you put ASIA ? it's not in EU, CA or US

File: packages/super-components/utils/src/url.ts [hunk 2]
```diff
@@ -1,18 +1,75 @@
...
+
```
[the remark this answers is not in the document]
Comment (chipp972) [reply packages/super-components/utils/src/url.ts:34]: Ok thanks done

File: packages/super-components/utils/src/url.ts (hunk 1 above)
[the remark this answers is not in the document]
Comment (chipp972) [reply packages/super-components/utils/src/url.ts:33]: Ok it is in CA in the container url-constants file

File: packages/super-components/components/msc-billing-tile/src/msc-billing-tile/msc-billing-renewal.tsx [hunk 3]
```diff
@@ -138,18 +135,15 @@ export class MscBillingRenewal implements IMSCBillingRenewal {
...
-                serviceType: this.serviceType,
-              })}
+              href={this.urls?.manageRenewUrl}
               color={ODS_THEME_COLOR_INTENT.primary}
             >
               {this.localeStrings.billing_services_actions_menu_manage_renew}
             </osds-link>
             <osds-link
               data-tracking={this.anticipateRenewDataTracking}
               target="_blank"
```
Comment (Kranysys) [inline packages/super-components/components/msc-billing-tile/src/msc-billing-tile/msc-billing-renewal.tsx:145]: `OdsHTMLAnchorElementTarget._blank`

File: packages/super-components/components/msc-billing-tile/src/msc-billing-tile/msc-billing-renewal.tsx [hunk 4]
```diff
@@ -138,18 +135,15 @@ export class MscBillingRenewal implements IMSCBillingRenewal {
             <osds-link
               data-tracking={this.manageRenewDataTracking}
               target="_blank"
```
Comment (Kranysys) [inline packages/super-components/components/msc-billing-tile/src/msc-billing-tile/msc-billing-renewal.tsx:137]: Should change it to target={OdsHTMLAnchorElementTarget._blank}

File: packages/super-components/components/msc-billing-tile/src/msc-billing-tile/msc-billing-renewal.tsx (hunk 3 above)
Comment (chipp972) [reply packages/super-components/components/msc-billing-tile/src/msc-billing-tile/msc-billing-renewal.tsx]: done thanks

File: packages/super-components/components/msc-billing-tile/src/msc-billing-tile/msc-billing-renewal.tsx (hunk 4 above)
Comment (chipp972) [reply packages/super-components/components/msc-billing-tile/src/msc-billing-tile/msc-billing-renewal.tsx]: done thanks

**Review by Kranysys — APPROVED** [review]

**Pull request merged**